test/types/objectSearchCreate.test.ts

```diff
@@ -278,4 +267,25 @@ describe('objectSearchCreate', () => {
 
     expect(onSave).toHaveBeenCalledWith({ law_firm: results[0] });
   });
+
+  it('Handles onAddNewToggle', async () => {
+    const onAddNewToggle = jest.fn(),
+      { field, searchTerm, results, props } = getDefaults({ editProps: { onAddNewToggle } }),
+      tester = await getTester(props);
+
+    await tester.submit();
+    await tester.refresh();
```

Comment:
Similar to prior comments (and potentially applicable to lines 282-283 and 287-288, as well): do you need to `await` tester.submit() and then also do `tester.refresh()`?

Reply:
Yup, act overlap.